src/base/ReactorStructs.sol

```diff
@@ -34,19 +32,22 @@ struct OrderInfo {
     uint256 deadline;
 }
 
-/// @notice Every RelayOrder input is defined by a token, recipient,
-/// and amounts that define the start and end amounts on the decay curve.
-/// @dev These values are signed by the user. address(0) will set the recipient at run-time to the passed in feeRecipient value.
+/// @notice Every RelayOrder input is defined by a token, amount, and recipient,
+/// @dev These values are signed by the user
 struct Input {
     address token;
+    uint256 amount;
     address recipient;
-    uint256 startAmount;
-    uint256 maxAmount;
 }
 
-/// @notice Contains the final resolved transfer information. Amounts are decayed and feeRecipient is returned.
-struct ResolvedInput {
+/// @notice A RelayOrder can specify an increasing fee over time to be paid
+/// @dev The resolved amount will be sent to the passed in feeRecipient address
+struct FeeEscalator {
     address token;
-    address recipient;
-    uint256 amount;
+    uint256 startAmount;
+    uint256 maxAmount;
```

Comment:
nit: startAmount/endAmount or minAmount/maxAmount are more clear imo - start/max make it unclear that they're the two extremes in the escalator

Reply:
changed to start/end amount, good callout